iconservice/icon_service_engine.py

```diff
@@ -837,29 +838,46 @@ def _process_transaction(self,
 
         return score_address
 
-    def _get_byte_length(self, data) -> int:
+    def _get_data_size_in_bytes(self, revision: int, data) -> int:
         size = 0
-        if data:
-            if isinstance(data, dict):
-                for v in data.values():
-                    size += self._get_byte_length(v)
-            elif isinstance(data, list):
-                for v in data:
-                    size += self._get_byte_length(v)
-            elif isinstance(data, str):
-                # If the value is hexstring, it is calculated as bytes otherwise
-                # string
-                data_body = data[2:] if data.startswith('0x') else data
-                if is_lowercase_hex_string(data_body):
-                    size += ceil(len(data_body) / 2)
-                else:
-                    size += len(data.encode('utf-8'))
-            else:
-                # int and bool
-                if isinstance(data, int):
-                    size += byte_length_of_int(data)
+        if isinstance(data, dict):
+            size += self._get_dict_size_in_bytes(revision, data)
```

Comment:
Why is "+=" used instead of "="?
Though the both results are the same.

Reply:
"=" would be better.